com/server/src/main/java/com/database/service/DBService.java

```diff
@@ -0,0 +1,51 @@
+package com.database.service;
+
+import org.intellij.lang.annotations.Language;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class DBService {
+    private static final String DATABASE_URL = "jdbc:mysql://localhost:3306/web-app";
+    private static final String JDBC_DRIVER = "com.mysql.jdbc.Driver";
+    private static final String USER = "root";
+    private static final String PASSWORD = "root";
+
+    public void writeToDB (String login, String password, String email) {
+        try {
+            Class.forName(JDBC_DRIVER).newInstance();
+            try (Connection connection = DriverManager.getConnection(DATABASE_URL, USER, PASSWORD)) {
+                @Language("SQL")
+                String SQL = String.format("INSERT INTO users VALUES ('%s' , '%s' , '%s')",
```

Comment:
if we use prepare statement we can write:
```suggestion
                String SQL = String.format("INSERT INTO users VALUES (?, ?, ?)",
```